app/controllers/iiif_controller.rb

```diff
@@ -112,14 +114,28 @@ def image_info
       end
     end
 
+    ##
+    # When we are in a degraded state, we are giving back resized width/height
```

Comment:
I'd like to see a comment emphasizing how much of a terrible, implementation-specific hack this is.